docs/Docker.md

```diff
@@ -0,0 +1,36 @@
+# Docker
+
+## Developing in Docker
+
+If running for the first time, do:
+
+1. Delete local `node_modules` folder if it exists
```

Comment:
I would still add a `.dockerignore` file with node_modules, should someone forget this step

Reply:
I don't think that would do anything? As far as I can tell, the [`.dockerignore` file](https://docs.docker.com/engine/reference/builder/#dockerignore-file) is only relevant for the `ADD` and `COPY` operations, which we're not doing.